source/frontend/src/utils/mapPropertyUtils.ts

```diff
@@ -57,54 +57,53 @@ export const getPropertyName = (property: IMapProperty): PropertyName => {
   return { label: NameSourceType.NONE, value: '' };
 };
 
-export const getPrettyLatLng = (location?: Api_Geometry) =>
+export const getPrettyLatLng = (location: ApiGen_Concepts_Geometry | undefined | null) =>
   compact([
     location?.coordinate?.x?.toFixed(6) ?? 0,
     location?.coordinate?.y?.toFixed(6) ?? 0,
   ]).join(', ');
 
-export const getLatLng = (location?: Api_Geometry | null): LatLngLiteral | null => {
+export const getLatLng = (
+  location: ApiGen_Concepts_Geometry | undefined | null,
+): LatLngLiteral | null => {
   const coordinate = location?.coordinate;
-  if (coordinate !== null && coordinate !== undefined) {
-    return { lat: coordinate.y!, lng: coordinate.x! };
+  if (exists(coordinate)) {
+    return { lat: coordinate.y, lng: coordinate.x };
   }
   return null;
 };
 
 export const getFilePropertyName = (
-  fileProperty?: Api_PropertyFile,
+  fileProperty: ApiGen_Concepts_FileProperty | undefined | null,
   skipName: boolean = false,
 ): PropertyName => {
-  if (fileProperty === undefined) {
+  if (!exists(fileProperty)) {
     return { label: NameSourceType.NONE, value: '' };
   }
 
-  if (
-    fileProperty.propertyName !== undefined &&
-    fileProperty.propertyName !== null &&
-    fileProperty.propertyName !== '' &&
-    skipName === false
-  ) {
+  if (exists(fileProperty.propertyName) && fileProperty.propertyName !== '' && skipName === false) {
     return { label: NameSourceType.NAME, value: fileProperty.propertyName ?? '' };
-  } else if (!!fileProperty.property) {
+  } else if (exists(fileProperty.propertyName)) {
```

Comment:
this looks like a functional change to me.

Reply:
yep, it was a mistake. A later commit fixes it